syft/serde/protobuf/torch_serde.py

```diff
@@ -0,0 +1,238 @@
+"""
```

Comment:
Also seem to have common code with msgpack/torch_serde.py.
I think it's fine for now and can be addressed later if needed.

Reply:
Yeah, this is "get it working at all" code with no attempt to make it pretty or well-factored. This PR is (generously) much closer to 20% complete than 80% complete, so I'm a lot more worried about getting the core concepts right than polishing at this point.

Reply:
#2869 extracts some of the shared code into a separate Torch serde. Possible that it can be further refactored to make use of common abstractions, but that at least removes direct duplication.